src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(Color color, int side) {
```

Comment:
Consider creating separate variables for the 'side' value before passing it to the constructor for better readability, especially if the value is being generated or calculated.